RadarSDK/RadarAPIClient.m

```diff
@@ -133,46 +133,20 @@ - (void)trackWithLocation:(CLLocation * _Nonnull)location
     params[@"deviceType"] = [RadarUtils deviceType];
     params[@"deviceMake"] = [RadarUtils deviceMake];
     params[@"sdkVersion"] = [RadarUtils sdkVersion];
-    NSString *deviceModel = [RadarUtils deviceModel];
-    if (deviceModel) {
-        params[@"deviceModel"] = deviceModel;
-    }
-    NSString *deviceOS = [RadarUtils deviceOS];
-    if (deviceOS) {
-        params[@"deviceOS"] = deviceOS;
-    }
-    NSString *country = [RadarUtils country];
-    if (country) {
-        params[@"country"] = country;
-    }
-    NSNumber *timeZoneOffset = [RadarUtils timeZoneOffset];
-    if (timeZoneOffset) {
-        params[@"timeZoneOffset"] = timeZoneOffset;
-    }
-    NSString *uaChannelId = [RadarUtils uaChannelId];
-    if (uaChannelId) {
-        params[@"uaChannelId"] = uaChannelId;
-    }
-    NSString *uaNamedUserId = [RadarUtils uaNamedUserId];
-    if (uaNamedUserId) {
-        params[@"uaNamedUserId"] = uaNamedUserId;
-    }
-    NSString *uaSessionId = [RadarUtils uaSessionId];
-    if (uaSessionId) {
-        params[@"uaSessionId"] = uaSessionId;
-    }
+    params[@"deviceModel"] = [RadarUtils deviceModel];
+    params[@"deviceOS"] = [RadarUtils deviceOS];
+    params[@"country"] = [RadarUtils country];
+    params[@"timeZoneOffset"] = [RadarUtils timeZoneOffset];
+    params[@"uaChannelId"] = [RadarUtils uaChannelId];
+    params[@"uaNamedUserId"] = [RadarUtils uaNamedUserId];
+    params[@"uaSessionId"] = [RadarUtils uaSessionId];
     params[@"source"] = [Radar stringForSource:source];
     
     NSString *host = [RadarSettings host];
     NSString *url = [NSString stringWithFormat:@"%@/v1/track", host];
     url = [url stringByAddingPercentEncodingWithAllowedCharacters:[NSCharacterSet URLQueryAllowedCharacterSet]];
     
-    NSDictionary *headers = @{
-        @"Authorization": publishableKey,
-        @"Content-Type": @"application/json",
-        @"User-Agent": [RadarUtils userAgent],
-        @"X-Radar-Config": @"true",
-    };
+    NSDictionary *headers = [RadarAPIClient headersWithPublishableKey:publishableKey];
```

Comment:
so, it seems like we're going to be sending certain params (ex. `deviceModel`, `deviceOS`, etc.) in the request params as well as the headers? is that intended?

Reply:
Later we can pick these up from headers server-side, but keeping in the body for backcompat right now